vshn/website
# feat(components): implement team filters and display author description with translation

**Describe what changes this pull request brings**
- This pull request brings the filters for Team Members


**Describe any additional changes**
- adjust component PartnersList
- display author description with translation on Blog Post page
- add sitemap generation


**Steps to test**
1. Open [the page](https://team-filters--vshn.netlify.app/team/)
2. Confirm that everything looks and works as expected


**Screenshots**
![image](https://user-images.githubusercontent.com/48465000/107501746-7a8e0800-6ba0-11eb-94e9-c90b2beba432.png)



**Checklist**



**TODO**


**References**
- Close the task [Implement filtering for Team members just like on the Clients page](https://www.notion.so/Implement-filtering-for-Team-members-just-like-on-the-Clients-page-2678be50bd99450b9a9fdf6162326311)
